distributed/distributed.yaml

```diff
@@ -165,6 +165,7 @@ distributed:
          min: 1s  # the first non-zero delay between re-tries
          max: 20s  # the maximum delay between re-tries
     compression: auto
+    shard: 64MiB
```

Comment:
nitpick but the previous value was `64MB` not `64MiB` :)

Reply:
> I think that we should default to powers of two in genera

I guess it doesn't really matter but with powers of two we probably have the best chance to hit some kind of sweet mem alignment/cache size/whatever optimization so I'm all for it. 

However, if you prefer powers of two, you should used `MB`, shoudn't you? (I understand if you didn't want to push changes any more, just wondering if I messed something up in my mind :) )

```
MB == 1024 ** 2 / 2 ** 20
MiB == 1000 ** 2 / 10**6
```

Reply:
From the internet, the source of all truth.

Megabytes (MB) or Mebibytes (MiB)?

Though the article refers to Linux, the topic is applicable to all computers. ... According to these standards, technically a megabyte (MB) is a power of ten, while a mebibyte (MiB) is a power of two, appropriate for binary machines. A megabyte is then 1,000,000 bytes.Dec 23, 2001

Reply:
wow, I was wrong all those years. I remember a first semester class where my prof introduced the unit and I would've sworn all these years it was the other way round 🤦

Reply:
Maybe the prof was wrong too?  Imagine how they feel :)

Reply:
If he was wrong, he'd **never** admit to it!